src/dist.config.ts

```diff
@@ -128,7 +128,7 @@ function webpackConfig(args: any): webpack.Configuration {
 		...output,
 		path: outputPath,
 		chunkFilename: '[name].[chunkhash].bundle.js',
```

Comment:
do you want to drop the `chunkhash` from here too?

Reply:
Yes I do! I was testing with --single-bundle and didn't really know what `chunkFilename` was.